lib/diagnostic.rb

```diff
@@ -56,7 +67,13 @@ def example_params
   end
 
   describe 'POST create' do
-    # your test(s) here
+    before(:each) do
+      post :create, format: :json
+
+    it 'is succesful and returns JSON object'
+
+    expect(response.status).to eq(201)
+    expect(JSON.parse(response.body)).not_to be_nill
   end
```

Comment:
Be careful with your formatting here and make sure with a POST request, you have data to actually POST.

```
  describe 'POST create' do
    before(:each) do
      post :create, example: example_params, format: :json
    end

    it 'is successful' do
      expect(response).to be_successful
    end

    it 'renders a JSON response' do
      example_response = JSON.parse(response.body)
      expect(example_response).not_to be_nil
    end
  end
end
```